query-node/joystream-query-node/mappings/members.ts

```diff
@@ -1,3 +1,32 @@
-export function handleMemberRegistered() {
-	console.log(handleMemberRegistered.name);
+import { Memberships } from '../generated/indexer/entities/Memberships';
+import { DB } from '../generated/indexer';
+
+export async function handleMemberRegistered(db: DB) {
+  // Get event data
+  const { AccountId, MemberId } = db.event.event_params;
+
+  let member = new Memberships({ accountId: AccountId.toString(), memberId: +MemberId });
```

Comment:
- Why is `+MemberId` used?
- What type is `MemberId`, given that its `u64`, which can encode a larger integer than `Number` in typescript, which is a signed double-precision 64-bit, with `MAX_SAFE_INTEGER = 9007199254740991`.

Reply:
May be can use BigInt here? [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt). Typescript support it and `MAX_SAFE_INTEGER` is the same.

Reply:
It is polkadot.js Codec type. From our last meeting: it would be better by using typeorm's `ColumnOptions.transformer` to convert the value automatically with `to/from` functions.